ts/webui/src/components/experiment/trialdetail/chart/Para.tsx

```diff
@@ -56,9 +56,9 @@ class Para extends React.Component<ParaProps, ParaState> {
             customizeColumnsDialogVisible: false,
             availableDimensions: [],
             chosenDimensions:
-                localStorage.getItem('paraColumns') !== null
+                sessionStorage.getItem('paraColumns') !== null
```

Comment:
If I run a different experiment on the same port, would the storage still be persistent?

Reply:
Each tab page you open holds its own data